docs/Way-of-Working/Offboarding.md

```diff
@@ -0,0 +1,30 @@
+# Off Boarding
```

Comment:
Should we add a section about one of the following items:

1. Kubernetes / Digilab
2. Pleio community
3. Signal groups